### PR TITLE
Open safe app drawer

### PR DESCRIPTION
**Pull request by yagopv**
## What it solves

Resolves https://github.com/safe-global/web-core/issues/1414

## How this PR fixes it
Add Open Safe App drawer to the new design implementation

## Design
https://www.figma.com/file/ptTs6lDBeUuLNySroJ5PiF/Web-Master-File?node-id=2231%3A64547&t=zAlaQEiEdgkpk1P8-0

## Screenshots
image

### CODE REVIEW

File: src/components/new-safe-apps/SafeAppPreviewDrawer/SafeAppPreviewDrawer.tsx [hunk 1]
```diff
@@ -4,32 +4,85 @@ import Drawer from '@mui/material/Drawer'
...
 import Button from '@mui/material/Button'
+import { IconButton, SvgIcon } from '@mui/material'
+import Tooltip from '@mui/material/Tooltip'
 import type { SafeAppData } from '@safe-global/safe-gateway-typescript-sdk'
 
 import { getSafeAppUrl } from '@/components/new-safe-apps/SafeAppCard/SafeAppCard'
+import ChainIndicator from '@/components/common/ChainIndicator'
+import SafeAppIconCard from '../SafeAppIconCard/SafeAppIconCard'
+import SafeAppActionButtons from '../SafeAppActionButtons/SafeAppActionButtons'
+import CloseIcon from '@/public/images/common/close.svg'
```
Comment (DaniSomoza) [inline src/components/new-safe-apps/SafeAppPreviewDrawer/SafeAppPreviewDrawer.tsx:15]: Maybe we can use the [Material UI Close icon](https://mui.com/material-ui/material-icons/?query=close&selected=Close)?

**Comment by katspaugh** [comment]
Could you please remove the uppercase in these places? It should be "All apps", "My custom apps", "Open app" etc.

Screenshot 2023-01-13 at 09 33 16
Screenshot 2023-01-13 at 09 33 12

**Comment by DaniSomoza** [comment]
> Could you please remove the uppercase in these places? It should be "All apps", "My custom apps", "Open app" etc.
> 
> Screenshot 2023-01-13 at 09 33 16
> 

This can be done in this PR

> Screenshot 2023-01-13 at 09 33 12

I changed the tabs label in the parent branch to avoid conflicts ([new-safe-apps-design](https://github.com/safe-global/web-core/tree/new-safe-apps-design))

File: src/components/new-safe-apps/SafeAppPreviewDrawer/SafeAppPreviewDrawer.tsx (hunk 1 above)
Comment (yagopv) [reply src/components/new-safe-apps/SafeAppPreviewDrawer/SafeAppPreviewDrawer.tsx:15]: I did it but as the other 2 icons are custom ones the close from material feels different and more padded so i need to play with viewBox or other styles to fit it

image

Then on hover the border-radius as well need to be moved 

image

... so finally decided to add the custom one as the others. If you prefer the material one i can try to fit it

File: src/components/new-safe-apps/SafeAppPreviewDrawer/SafeAppPreviewDrawer.tsx [hunk 2]
```diff
@@ -4,32 +4,85 @@ import Drawer from '@mui/material/Drawer'
...
       <Box className={css.drawerContainer}>
-        <Typography>{safeApp?.name}</Typography>
+        {/* Toolbar */}
+
+        {safeApp && (
+          <Box display="flex" justifyContent="right">
+            <SafeAppActionButtons safeApp={safeApp} isBookmarked={isBookmarked} onBookmarkSafeApp={onBookmark} />
+            <Tooltip title={`Close ${safeApp.name} preview`} placement="top">
+              <IconButton
+                aria-label="close"
```
[the remark this answers is not in the document]
Comment (yagopv) [reply src/components/new-safe-apps/SafeAppPreviewDrawer/SafeAppPreviewDrawer.tsx:42]: make sense

**Comment by JagoFigueroa** [comment]
Órale! Tested and verified on [this](https://github.com/safe-global/web-core/pull/1537) señora branch with the whole feature. Espectacular trabajo 😉

**Pull request merged**